app/src/main/java/com/sopt/now/jumpit/ui/searchResult/SearchResultFragment.kt

```diff
@@ -0,0 +1,102 @@
+package com.sopt.now.jumpit.ui.searchResult
+
+import android.os.Bundle
+import android.text.Spannable
+import android.text.SpannableString
+import android.text.style.ForegroundColorSpan
+import android.util.Log
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.activityViewModels
+import com.google.android.material.snackbar.Snackbar
+import com.sopt.now.jumpit.R
+import com.sopt.now.jumpit.data.remote.response.SearchResult
+import com.sopt.now.jumpit.databinding.FragmentSearchResultBinding
+import com.sopt.now.jumpit.ui.base.BindingFragment
+
+class SearchResultFragment :
+    BindingFragment<FragmentSearchResultBinding>(R.layout.fragment_search_result) {
+
+    private lateinit var searchResultAdapter: SearchResultAdapter
+    private val viewModel: SearchResultViewModel by activityViewModels()
+    private var bottomSheet: SearchCategoryDialog? = null
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        setupRecyclerView()
+        setupPositionChipClickListener()
+        observeSearchResult()
+    }
+
+    private fun setupRecyclerView() {
+        searchResultAdapter = SearchResultAdapter {
+            Snackbar.make(binding.root, it.title, Snackbar.LENGTH_SHORT).show()
+        }
+        binding.rvSearchResult.adapter = searchResultAdapter
+    }
+
+    private fun setupPositionChipClickListener() {
+        binding.chipSearchCategoryPosition.setOnClickListener {
+            showBottomSheet()
+        }
+    }
+
+    private fun showBottomSheet() {
+        if (bottomSheet == null || !bottomSheet!!.isAdded) {
+            bottomSheet = SearchCategoryDialog()
+            bottomSheet!!.show(parentFragmentManager, "SearchCategoryDialog")
+        }
+    }
+
+    private fun observeSearchResult() {
+        viewModel.searchResults.observe(viewLifecycleOwner) { searchResults ->
+            updateSearchResultCount(searchResults)
+            updateEmptyTextVisibility(searchResults)
+            searchResultAdapter.submitList(searchResults)
+        }
+    }
```

Comment:
```suggestion
    private fun observeSearchResult() {
        viewModel.searchResults.observe(viewLifecycleOwner) { 
            updateSearchResultCount(it)
            updateEmptyTextVisibility(it)
            searchResultAdapter.submitList(it)
        }
    }
```
it을 안쓰고 ->을 써주는 이유는 머인가요? 저 정말 궁금해여 🤣

Reply:
변수명을 명시해주고자 했으나...! 이 상황에서는 굳이일듯 싶네요!! 좋습니다 👍